capstone-frontend/src/components/ListPage.jsx

```diff
@@ -110,16 +111,29 @@ class ListPage extends Component {
           <div>
             <Row>
               <Col>
-                <h2 className="mt-4 ml-2">{this.state.bookList.name}</h2>
-              </Col>
-              <Col className="text-right">
-                <SearchBookModal
-                  objectId={this.props.match.params.id}
-                  update={this.updateListPage}
-                  putURL="/api/booklist"
-                  type="booklist"
-                  btnStyle="btn btn-primary mt-4 mr-2" />
+                <h2 className="ml-2">{this.state.bookList.name}</h2>
+                {this.state.bookList.userID === window.localStorage.getItem('userID') ?
+                  <h5 className="mb-1 ml-2 text-muted">Owner</h5>
+                  : 
+                  <h5 className="mb-1 ml-2 text-muted">Collaborator</h5>}
```

Comment:
nit: Use single quotations for string literals
Should be updated consistently in this entire file

Reply:
Ok